task/06-conditions-n-loops-tasks.js

```diff
@@ -373,7 +454,17 @@ function toNaryString(num, n) {
  *   ['/web/favicon.ico', '/web-scripts/dump', '/webalizer/logs'] => '/'
  */
 function getCommonDirectoryPath(pathes) {
-    throw new Error('Not implemented');
+    let str = '';
+
+    for (let i = 0; i < pathes[0].length; ++i) {
```

Comment:
what this for-loop does?

Reply:
The for-loop iterates over the first element of the array and adds i-element to the string

Reply:
Technically yes it does that, not the asnwer i was looking for. This loop doesn't do anything useful. 
You need to remove this loop and replace it with something else. Hint: the string is a **reference type** (https://codeburst.io/explaining-value-vs-reference-in-javascript-647a975e12a0)  and string is also **immutable** (https://gomakethings.com/immutability-in-javascript/#strings)